Objects/complexobject.c

```diff
@@ -143,6 +189,64 @@ _Py_c_quot(Py_complex a, Py_complex b)
 
     return r;
 }
+
+Py_complex
+_Py_cr_quot(Py_complex a, double b)
+{
+    Py_complex r = a;
+    if (b) {
+        r.real /= b;
+        r.imag /= b;
+    }
+    else {
+        errno = EDOM;
+        r.real = r.imag = 0.0;
```

Comment:
I don't remember now but don't we use another convention for the real and imaginary components when a failure happens? (I think you told me once that it's set to -1 or something like that). On the other hand, is this convention the one defined by C99? If so, just ignore my comment.